### ISSUE TITLE
Tie user to uid

### ISSUE THREAD

**Issue by digininja** [Raikia/FiercePhish#69]
From what Chris says, you probably already have this planned for v2, but you need a way to tie users to the uid generated when sending emails.

I've got it direct from the database for now.

**Comment by Raikia** [comment]
Yeah, the whole reporting module is something that was never created (which is terrible of me) because there were a lot of "lessons learned" from this first version.  I'm working on a v2.0 rewrite that will include full reporting modules and everything for this, plus credential harvesting, website hosting, etc.
